crates/ty_ide/src/semantic_tokens.rs

```diff
@@ -230,6 +227,11 @@ impl<'db> SemanticTokenVisitor<'db> {
         modifiers: SemanticTokenModifier,
     ) {
         let range = ranged.range();
+
+        if range.is_empty() {
+            return;
+        }
```

Comment:
VS Code logs a warning for empty ranges, saying that they're invalid. Empty tokens are also pretty useless because how do you style a token that has no representation in the source :)